js/views/inbox_view.js

```diff
@@ -156,6 +156,11 @@
             });
 
             var inboxCollection = getInboxCollection();
+
+            inboxCollection.on('messageSendErrors', function(){
```

Comment:
`s/messageSendError/messageError` to match the event triggered in conversations.js?